frontend/src/AboutPage.js

```diff
@@ -9,9 +9,10 @@ function AboutPage() {
 
```

Comment:
in file above, "About.css" there is no code or content. can delete this

Reply:
Thanks, I'll delete that!